common/src/Assets/EntityDefinitionManager.h

```diff
@@ -51,8 +50,6 @@ namespace TrenchBroom {
             Cache m_cache;
 
             NotifierConnection m_notifierConnection;
```

Comment:
You can also drop `m_notifierConnection` and remove the corresponding include. It's no longer used.